test/integration/targets/eos_eapi/tests/cli/configure.yaml

```diff
@@ -21,13 +21,13 @@
 - name: Expect endpoint ports to be set
   assert:
     that:
-        - http_config.stdout[0].httpServer.port == 81
+        - http_config.stdout[0].httpServer.port == 80
```

Comment:
Yes, agreed we should provide a warning that the port will not be changed due to the protocol not being enabled.